test/cluster/cluster_helper.go

```diff
@@ -616,6 +617,45 @@ func validateMigrateFillDelay(
 	return err
 }
 
+// Helper function to get number of migrations in progress
+func getMigrationsInProgress(ctx goctx.Context, k8sClient client.Client,
+	clusterNamespacedName types.NamespacedName) (int, error) {
+	aeroCluster, err := getCluster(k8sClient, ctx, clusterNamespacedName)
+	if err != nil {
+		return -1, err
+	}
+
+	podList, err := getPodList(aeroCluster, k8sClient)
+	if err != nil {
+		return -1, err
+	}
+
+	maxMigrations := 0
+
+	for idx := range podList.Items {
+		pod := &podList.Items[idx]
+
+		asinfo, err := getASInfo(logger, k8sClient, ctx, clusterNamespacedName, pod.Name, "statistics")
```

Comment:
cross-check how to call getASInfo() function, last argument "statistics" passsed here is not correct.